packages/oui-button/README.md

```diff
@@ -0,0 +1,73 @@
+# Button
+
+<component-status cx-design="complete" ux="prototype"></component-status>
+
+## Usage
+
+```html
+<oui-button
+  text="..."
+  variant="[primary|secondary|link]"
+  variant-nav="[previous|next]"
+  type="[submit|button|reset]"
+  id="..."
+  name="..."
+  disabled="..."
+  on-click="..."
+></oui-button>
+```
+
+## Examples
+
+### Default
+
+```html:preview
+<oui-button text="Default button" on-click="click()"></oui-button>
+```
+
+### Action buttons
+
+```html:preview
+<div>
+  <oui-button variant="primary" text="Primary"></oui-button>
+  <oui-button variant="secondary" text="Secondary"></oui-button>
+  <oui-button variant="link" text="Link"></oui-button>
+</div>
+
+<div>
+  <oui-button variant="primary" text="Primary" disabled></oui-button>
+  <oui-button variant="secondary" text="Secondary" disabled></oui-button>
+  <oui-button variant="link" text="Link" disabled></oui-button>
+</div>
+```
+
+### Steps buttons
+
+```html:preview
+<div>
+  <oui-button variant-nav="previous" text="Previous" on-click="previous()"></oui-button>
+  <oui-button variant="primary" variant-nav="next" text="Next" on-click="next()"></oui-button>
+</div>
+
+<div>
+  <oui-button variant-nav="previous-step" text="Previous" on-click="previous()" disabled></oui-button>
+  <oui-button variant="primary" variant-nav="next" text="Next" on-click="next()" disabled></oui-button>
+</div>
+```
+
+### Accessibility
```

Comment:
I think it should be at the same level than `Examples` and `API`, not under `Examples`.

Reply:
This part will be removed, because it'll have its own attribute.

Reply:
I added an example to this part. I think it can stay in the examples area.
I move up the API part next to Usage.